superset-frontend/src/pages/DatasetList/index.tsx

```diff
@@ -181,6 +186,11 @@ const DatasetList: FunctionComponent<DatasetListProps> = ({
     setSSHTunnelPrivateKeyPasswordFields,
   ] = useState<string[]>([]);
 
+  const PREVENT_UNSAFE_DEFAULT_URLS_ON_DATASET = useSelector<
+    any,
```

Comment:
Could we use a more strict typing here? Like `ViewState` from `src/views/types.ts` (you'd just need to add `PREVENT_UNSAFE_DEFAULT_URLS_ON_DATASET: boolean` to it)